index.html

```diff
@@ -6142,7 +5294,7 @@ <h2 id="AudioWorklet-Examples">
           This is an <code>Event</code> object which is dispatched to
           <a><code>ScriptProcessorNode</code></a> nodes. It will be removed
           when the ScriptProcessorNode is removed, as the replacement
-          <a>AudioWorker</a> uses the <a>AudioProcessEvent</a>.
+          <a>AudioWorkletNode</a> uses a different approach.
```

Comment:
Not for this PR, but I think everything would read much better if we just removed such explanations.  We've already marked ScriptProcessorNode as deprecated; no need to keep saying AudioWorkletNodes do something else.